README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
+
 [![npm version](https://badge.fury.io/js/lint-prepush.svg)](https://www.npmjs.com/package/lint-prepush)
 [![npm downloads](https://img.shields.io/npm/dt/lint-prepush.svg)](https://www.npmtrends.com/lint-prepush)
 [![GitHub license](https://img.shields.io/github/license/theenadayalank/lint-prepush.svg)](https://github.com/theenadayalank/lint-prepush/blob/master/LICENSE)
 [![Build Status](https://travis-ci.org/theenadayalank/lint-prepush.svg?branch=master)](https://travis-ci.org/theenadayalank/lint-prepush)
 
+> Run linters on committed files of a GIT Branch
 
-> Run linters on committed files of a GIT Branch🔬
-
-## Getting Started 🔮
+## Getting Started
 
 This package will run linters on your project for the committed files in your branch.
 
-### Prerequisites🔭
-
-It require Node.js v6 or newer. It also requires a package to manage git hooks. I strongly suggest [Husky](https://github.com/typicode/husky) which I use for most of my projects.
-
-
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm test",
-      "pre-push": "npm test",
-      "...": "..."
-    }
-  }
-}
-```
-
-### Installing
+### Prerequisites
 
-* This project requires a package to manage git hooks depending. 
-* I strongly suggest [Husky](https://github.com/typicode/husky) which pauses the git push by overriding pre-commit hook (it almost overrides all hooks, here we need only pre-push hook) and allow us to run our custom scripts and resumes pushing.
+- Node.js `>=6.0.0`
+- a separate package to manage integration with git hooks, like [husky](https://github.com/typicode/husky)
 
+### Installation
 
-### npm
+Install `lint-prepush` and `husky` via [`npm`](https://www.npmjs.com/):
 
-```bash
+```shell
 npm install --save-dev husky lint-prepush
 ```
 
-or using [`yarn`](https://yarnpkg.com/):
+You can also use [`yarn`](https://yarnpkg.com/) to install them:
 
-```bash
+```shell
 yarn add --dev husky lint-prepush
 ```
 
 ### Usage
 
-Configure the following scripts in package.json to lint your committed files 🔧. You can also follow any of the  [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) methods to configure lint-prepush.
+Add the following to your `package.json` to lint your committed files. You can also
+follow any of the [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) methods
+to configure `lint-prepush`.
 
-```diff
+```json
 {
-  "scripts": {
-+   "prepush": "lint-prepush"
+  "husky": {
+    "hooks": {
+      "pre-push": "lint-prepush",
+    }
   },
-+ "lint-prepush": {
-+   "base": "master",
-+    "tasks": {
-+      "*.js": [
-+        "eslint"
-+      ]
-+    }
-+  }
+  "lint-prepush": {
+    "base": "master",
+    "tasks": {
+      "**/*.js": [
+        "npx eslint"
```

Comment:
need for npx?

Reply:
If you don't install `eslint` globally and are running it outside of a script that already adds your local `node_modules` to your `$PATH`, you have to reference ESLint via `./node_modules/.bin/eslint` like in the `package.json` for `lint-prepush`.

Using `npx` just means that you can reference it as `eslint` and let it resolve to the local version. It will also get `eslint` from npm if the project doesn't have it as a dependency and it isn't installed locally yet.

https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner

Reply:
Cool 😃

Reply:
Can this be implemented [here](https://github.com/theenadayalank/lint-prepush/blob/master/package.json#L19) ?

Reply:
Yeah, it would work there. I thought about adding it as part of this PR, but it seemed like that might be better as its own PR.

Reply:
That's Great 👍